base_layer/wallet/tests/transaction_service/service.rs

```diff
@@ -663,6 +664,186 @@ fn single_transaction_to_self() {
     });
 }
 
+#[test]
+fn send_one_sided_transaction_to_other() {
+    let mut runtime = create_runtime();
+
+    let factories = CryptoFactories::default();
+    // Alice's parameters
+    let alice_node_identity = Arc::new(
+        NodeIdentity::random(&mut OsRng, get_next_memory_address(), PeerFeatures::COMMUNICATION_NODE).unwrap(),
+    );
+
+    // Bob's parameters
+    let bob_node_identity = Arc::new(
```

Comment:
Copy paste?
We have Bob here, but bob is never used?

Reply:
It is used in line 689 and 728; Alice sending to Bob.

Reply:
689 is a log message
But I see, 728 is sending to his pubkey. Missed that one. 

We should try and let Bob spend this tx. Then we know its valid. 
We only currently know its constructed. Not if its correctly constructed.

Reply:
That is for the next sprint.

Reply:
Agreed; ROI will not be worth it without the receiver bits implemented.